addons/l10n_ro_edi/views/ciusro_document_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="l10n_gr_mydata_document_form" model="ir.ui.view">
+        <field name="name">ciusro.document.form</field>
+        <field name="model">ciusro.document</field>
```

Comment:
You shouldn't have views on this document. Imo, it's technical stuff and all user-friendly stuff should be on the invoice. For documents, do the same as Mexico: add a download button on the documents